src/Core/MailTemplate/FolderThemeScanner.php

```diff
@@ -45,6 +45,19 @@ final class FolderThemeScanner
      */
     private $baseThemeFolder;
 
+    /**
+     * @var string
+     */
+    private $moduleDirectory;
+
+    /**
+     * @param string $moduleDirectory
+     */
+    public function __construct($moduleDirectory)
+    {
+        $this->moduleDirectory = $moduleDirectory;
```

Comment:
Can you document this as a BC break?

Reply:
I think this is not a BC break ;) as this is a class only used as a Symfony service and we authorized such changes

> Symfony service constructors
Since classes whose instantiation is delegated to the Symfony service container are not supposed to be instantiated individually, changing the constructor of those classes is allowed.

https://github.com/PrestaShop/ADR/blob/master/0017-backward-compatibility-promise.md#symfony-service-constructors

Reply:
You're right, I always forget this one 😅